### PR TITLE
#564 test result page changes

### PR DESCRIPTION
**Pull request by cescoferraro**
This PR addreses issues raised by #564 

## Changes

Test are at bottom of test result should be open when you first come to the test. Currently it is closed and difficult to notice.
Show a chevron on the test result bar to indicate it can be opened & closed
Show 'Timeline view' and 'Graph view' mouseover on the buttons for switching view.
Delete assertion & Edit assertion tooltips & change delete to a trashcan icon
Remove the #de22 label from span label

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by kdhamric** [comment]
If you can use Loom to create really quick demos, it will help me when looking at the changes to make sure everything looks right.

**Comment by cescoferraro** [comment]
https://www.loom.com/share/30585d33cfab4bddab968b6ab87bbd01

File: web/package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "start": "craco start",
+    "dev": "concurrently 'npm run start' 'npm run port-forward'",
```
Comment (xoscar) [inline web/package.json:44]: great!

File: web/src/components/AssertionCard/AssertionCard.styled.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const SelectorListText = styled(Typography.Text).attrs({
 `;
 
 export const SpanCountText = styled(Typography.Text)`
-  font-size: 12;
```
Comment (xoscar) [inline web/src/components/AssertionCard/AssertionCard.styled.ts:33]: lol good catch

File: web/src/components/DiagramSwitcher/DiagramSwitcher.tsx [hunk 3]
```diff
@@ -9,18 +10,24 @@ interface IProps {
   selectedType: SupportedDiagrams;
 }
 
+const color = '#FFFFFF';
```
Comment (xoscar) [inline web/src/components/DiagramSwitcher/DiagramSwitcher.tsx:13]: Let's think about incorporating the style components theme so we don't have to hardcode colors moving forward (future work)

File: web/src/components/TraceDrawer/TraceDrawerHeader.tsx [hunk 4]
```diff
@@ -80,6 +80,9 @@ const TraceDrawerHeader: React.FC<IProps> = ({
         >
           Add Assertion
         </S.AddAssertionButton>
+        <span style={{marginLeft: 16}}>
```
Comment (xoscar) [inline web/src/components/TraceDrawer/TraceDrawerHeader.tsx:83]: We could or not, create styled-components everytime we try to style an HTML/antd tag, let's think about that for the future.

**Review by xoscar — APPROVED** [review]
Hey @cescoferraro good job! I left some non-blocking comments

**Pull request merged**